ReflectorIOS/Helpers/Theme.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Theme.swift
+//  ReflectorIOS
+//
+//  Created by Nikhil Menon on 9/14/20.
+//  Copyright © 2020 Nikhil Menon. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
```

Comment:
Remove one extra whitespace. You may have a maximum of one whitespace line between code.